Berksfile.lock

```diff
@@ -123,7 +142,7 @@ GRAPH
   python (1.4.6)
     build-essential (>= 0.0.0)
     yum-epel (>= 0.0.0)
-  rabbitmq (3.3.0)
+  rabbitmq (3.9.0)
```

Comment:
What was the error?

On Tue, Feb 10, 2015, 8:13 AM jmetzmeier <notifications@github.com> wrote:

> In Berksfile.lock
> <https://github.com/pythian/opsviz/pull/31#discussion_r24414003>:
>
> > @@ -123,7 +142,7 @@ GRAPH
> >    python (1.4.6)
> >      build-essential (>= 0.0.0)
> >      yum-epel (>= 0.0.0)
> > -  rabbitmq (3.3.0)
> > +  rabbitmq (3.9.0)
>
> This version of the rabbitmq did not work on the version of chef opsworks
> runs when I tried it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pythian/opsviz/pull/31/files#r24414003>.
>